Декоративные элементы

Yarik-333 opened the pull request:
---
:mortar_board: [Декоративные элементы](https://up.htmlacademy.ru/adaptive/27/user/2179843/tasks/9)

:boom: https://htmlacademy-adaptive.github.io/2179843-sedona-27/5/

Yarik-333 committed:
Добавляет адаптивные декоративные элементы

Yarik-333 committed:
исправляет ошибки

Yarik-333 committed:
Исправляет ошибки

Yarik-333 committed:
исправляет ошибки

vadimrahmanov commented on source/js/script.js:
```diff
@@ -0,0 +1,14 @@
+let navMain = document.querySelector('.main-nav');
+let navToggle = document.querySelector('.main-nav__toggle');
+
```
let можно заменить на const, но это не обязательно.

vadimrahmanov commented on source/less/blocks/catalog/gallery.less:
```diff
@@ -71,6 +71,20 @@
 .gallery__item {
+  position: relative;
+
+  @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/catalog/video.less:
```diff
@@ -8,6 +9,17 @@
   margin-top: 17px;
   margin-bottom: 38px;
+
+  @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/form/registration.less:
```diff
@@ -4,19 +4,87 @@
+    }
+  }
+
+  @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/index/advantages.less:
```diff
@@ -33,15 +39,44 @@
+    flex-direction: column;
+  }
+
+  @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/main-header.less:
```diff
@@ -1,6 +1,41 @@
+  }
+}
+
+@media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/main-nav.less:
```diff
@@ -1,12 +1,54 @@
   }
 }
+
+@media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/site-list.less:
```diff
@@ -33,16 +37,28 @@
       color: #766357;
     }
+
+    @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/site-list.less:
```diff
@@ -51,8 +67,29 @@
   padding-bottom: 12px;
   display: block;
 
+  @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/index.html:
```diff
@@ -13,7 +13,7 @@
       <img class="main-header__logo-image" src="img/logo.svg" width="185" height="96" alt="Логотип Sedona">
     </a>
-    <nav class="main-header__nav main-nav">
+    <nav class="main-header__nav main-nav main-nav--opened main-nav--nojs">
```
Если включён JS, то по-умолчанию при открытии страницы на мобильном устройстве меню должно быть закрыто, а у тебя открыто. Это касается всех страниц.

vadimrahmanov commented on source/form.html:
```diff
@@ -62,7 +62,7 @@ <h2 class="visually-hidden">Форма отзыва</h2>
+            <h3 class="registration__title registration__title--divider">Представьтесь:</h3>
             <div class="registration__wrapper">
               <label class="registration__label" for="input-name">ИМЯ*:</label>
               <input class="registration__input" type="text" name="name" id="input-name" placeholder="Пётр">
```
Это не плейсхолдеры, а value

vadimrahmanov commented on source/form.html:
```diff
@@ -78,16 +78,16 @@ <h3 class="registration__title">Представьтесь:</h3>
             </div>
```
Это не плейсхолдеры, а value

vadimrahmanov commented on source/js/script.js:
```diff
@@ -0,0 +1,14 @@
+  } else {
+    navMain.classList.add('main-nav--closed');
+    navMain.classList.remove('main-nav--opened');
+  }
```
Здесь лучше в целом использовать `classList.toggle`

vadimrahmanov commented on source/less/blocks/form/comments.less:
```diff
@@ -1,16 +1,24 @@
   font-weight: 700;
+  font-size: 18px;
+
+  @media (max-width: @tablet-width) {
```
Если у тебя mobile-first, то везде должны `min-width`.

vadimrahmanov commented on source/less/blocks/main-header.less:
```diff
@@ -1,6 +1,41 @@
+}
+
+.main-header__logo-image {
+  z-index: 1;
```
z-index только работает на элементах с позиционированием.

vadimrahmanov commented on source/less/blocks/site-list.less:
```diff
@@ -14,17 +15,20 @@
-  display: flex;
-  justify-content: center;
+  display: block;
   font-family: "PT Sans", sans-serif;
```
Шрифт лучше в переменную загнать

Yarik-333 committed:
Исправляет ошибки

1) без z-index края логотипа обрезаются, можно его оставить?
2) скрипт полностью копировал с демки, не хочу менять его, а то я пока не очень его понимаю.

Yarik-333 committed:
Исправляет ошибки

Yarik-333 committed:
Исправляет ошибки
